### PR TITLE
UseAnimations affects flyouts as well

### PR DESCRIPTION
**Pull request by quark-zju**
Flyouts animation is less annoying but since we have the option, this is more consistent.

I was updating #1622 but found it had been closed...

### CODE REVIEW

**Comment by Epix37** [comment]
Anything else before I merge this? :P

File: Hearthstone Deck Tracker/FlyoutControls/Options/Tracker/TrackerAppearance.xaml.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public void Load()
 			ComboBoxDeckLayout.ItemsSource = Enum.GetValues(typeof(DeckLayout));
 			ComboBoxIconSet.ItemsSource = Enum.GetValues(typeof(IconStyle));
 			CheckboxDeckPickerCaps.IsChecked = Config.Instance.DeckPickerCaps;
-			CheckboxUseDialogAnimation.IsChecked = Config.Instance.UseDialogAnimations;
+			CheckboxUseDialogAnimation.IsChecked = Config.Instance.UseAnimations;
```
Comment (Epix37) [inline Hearthstone Deck Tracker/FlyoutControls/Options/Tracker/TrackerAppearance.xaml.cs]: this needs to be CheckboxUseAnimations

Comment (quark-zju) [inline Hearthstone Deck Tracker/FlyoutControls/Options/Tracker/TrackerAppearance.xaml.cs]: Nice catch! I will update it soon.

**Comment by quark-zju** [comment]
I think it's okay for merge now :)

**Pull request merged**